src/utils/pgwire/src/pg_server.rs

```diff
@@ -143,12 +155,13 @@ mod tests {
     struct MockSession {}
 
     #[async_trait::async_trait]
-    impl Session for MockSession {
+    impl Session<BoxStream<'static, RowSetResult>> for MockSession {
         async fn run_statement(
             self: Arc<Self>,
             sql: &str,
             _format: bool,
-        ) -> Result<PgResponse, Box<dyn Error + Send + Sync>> {
+        ) -> Result<PgResponse<BoxStream<'static, RowSetResult>>, Box<dyn Error + Send + Sync>>
```

Comment:
Can we also get rid of the BoxStream here?

Reply:
Is performance important for mocked session 🤣 ?

Reply:
Oops, my bad 🤣